server.go

```diff
@@ -1497,29 +1497,14 @@ func newServer(cfg *Config, listenAddrs []net.Addr,
 
 	if cfg.WtClient.Active {
 		policy := wtpolicy.DefaultPolicy()
+		policy.MaxUpdates = cfg.WtClient.MaxUpdates
 
-		if cfg.WtClient.SweepFeeRate != 0 {
-			// We expose the sweep fee rate in sat/vbyte, but the
-			// tower protocol operations on sat/kw.
-			sweepRateSatPerVByte := chainfee.SatPerKVByte(
-				1000 * cfg.WtClient.SweepFeeRate,
-			)
-			policy.SweepFeeRate = sweepRateSatPerVByte.FeePerKWeight()
-		}
-
-		if cfg.WtClient.MaxUpdates != 0 {
-			policy.MaxUpdates = cfg.WtClient.MaxUpdates
-		}
-
-		sessionCloseRange := uint32(wtclient.DefaultSessionCloseRange)
-		if cfg.WtClient.SessionCloseRange != 0 {
-			sessionCloseRange = cfg.WtClient.SessionCloseRange
-		}
-
-		maxTasksInMemQueue := uint64(wtclient.DefaultMaxTasksInMemQueue)
-		if cfg.WtClient.MaxTasksInMemQueue != 0 {
-			maxTasksInMemQueue = cfg.WtClient.MaxTasksInMemQueue
-		}
+		// We expose the sweep fee rate in sat/vbyte, but the tower
+		// protocol operations on sat/kw.
+		sweepRateSatPerVByte := chainfee.SatPerKVByte(
+			1000 * cfg.WtClient.SweepFeeRate,
+		)
+		policy.SweepFeeRate = sweepRateSatPerVByte.FeePerKWeight()
```

Comment:
So now the user could set these values to 0 (which wasn't possible before), so I wonder if we need some validation to avoid incorrect settings?

Reply:
good point! do you think it is better to:
1. add a validation func that returns an error if any setting is set to 0, or
2. add a validation func that sets the value back to the defaults if a 0 value is encountered?

option 2 is kinda how it was before but silently ignores the user if they misuse the config options. So i'm leaning towards option 1

Reply:
Yeah, I think option 1 is better.

Reply:
cool - added!